src/testRunner/unittests/tsbuild/outFile.ts

```diff
@@ -192,108 +125,77 @@ namespace ts {
-            builder.build();
-            host.assertDiagnosticMessages(getExpectedDiagnosticForProjectsInBuild(relSources[Project.first][Source.config], relSources[Project.second][Source.config], relSources[Project.third][Source.config]),
-                [Diagnostics.Project_0_is_out_of_date_because_oldest_output_1_is_older_than_newest_input_2, relSources[Project.first][Source.config], relOutputFiles[Project.first][Ext.js], relSources[Project.first][Source.ts][Part.one]],
-                [Diagnostics.Building_project_0, sources[Project.first][Source.config]],
-                [Diagnostics.Project_0_is_up_to_date_because_newest_input_1_is_older_than_oldest_output_2, relSources[Project.second][Source.config], relSources[Project.second][Source.ts][Part.one], relOutputFiles[Project.second][Ext.js]],
-                [Diagnostics.Project_0_is_out_of_date_because_output_of_its_dependency_1_has_changed, relSources[Project.third][Source.config], "src/first"],
-                [Diagnostics.Building_project_0, sources[Project.third][Source.config]]
-            );
-            host.clearDiagnostics();
-            tick();
-
-            // Make incremental build with change in file that doesnt affect dts
-            appendText(fs, relSources[Project.first][Source.ts][Part.one], "console.log(s);");
-            builder = createSolutionBuilder(host, { verbose: true, incremental: true });
-            builder.build();
-            // Builds completely because tsbuildinfo is old.
-            host.assertDiagnosticMessages(
-                getExpectedDiagnosticForProjectsInBuild(relSources[Project.first][Source.config], relSources[Project.second][Source.config], relSources[Project.third][Source.config]),
-                [Diagnostics.Project_0_is_out_of_date_because_oldest_output_1_is_older_than_newest_input_2, relSources[Project.first][Source.config], relOutputFiles[Project.first][Ext.js], relSources[Project.first][Source.ts][Part.one]],
-                [Diagnostics.Building_project_0, sources[Project.first][Source.config]],
-                [Diagnostics.Project_0_is_up_to_date_because_newest_input_1_is_older_than_oldest_output_2, relSources[Project.second][Source.config], relSources[Project.second][Source.ts][Part.one], relOutputFiles[Project.second][Ext.js]],
-                [Diagnostics.Project_0_is_out_of_date_because_oldest_output_1_is_older_than_newest_input_2, relSources[Project.third][Source.config], relOutputFiles[Project.third][Ext.buildinfo], "src/first"],
-                [Diagnostics.Building_project_0, sources[Project.third][Source.config]]
-            );
-            host.clearDiagnostics();
+            commandLineArgs: ["--b", "/src/third", "--i", "--verbose"],
+            incrementalScenarios: [
+                {
+                    subScenario: "Make non incremental build with change in file that doesnt affect dts",
+                    buildKind: BuildKind.IncrementalDtsUnchanged,

```

Comment:
The line above says "non incremental" - is the disagreement intentional?

Reply:
I have removed the kind later in #48572